changes/tasks.py

```diff
@@ -198,87 +198,10 @@ def baby_switch(self, change):
         self.log.info("Starting switch to baby")
 
         self.log.info("Retrieving active subscriptions")
-        active_subs = sbm_client.get_subscriptions(
-            {"identity": change.registrant_id, "active": True}
-        )["results"]
```

Comment:
I'm a little bit confused as to why we're removing certain lines, changing the functionality, instead of just deleting the whole task.

Reply:
This task is needed by another `validate_implement`, I have tried removing it and the entire validate fails, so I opted for removing the part that is not needed in the task